src/billing-demo/Cargo.toml

```diff
@@ -4,6 +4,7 @@ description = "Microservice demo using Materialize to power a real-time billing
 version = "0.1.0"
 authors = ["Brandon W Maister <bwm@materialize.io>"]
 edition = "2018"
+publish = false
```

Comment:
should we add this to the lint script?

Reply:
It already is part of the lint script, in a way. Without `publish = false`, the missing license will trigger a `cargo deny` error.